src/ert/analysis/_es_update.py

```diff
@@ -49,16 +49,24 @@ class ErtAnalysisError(Exception):
 
 @dataclass
 class UpdateSnapshot:
-    obs_name: npt.NDArray[np.str_]
-    obs_value: npt.NDArray[np.float32]
-    obs_std: npt.NDArray[np.float32]
-    obs_mask: npt.NDArray[np.bool_]
-    response_mean: npt.NDArray[np.float32]
-    response_std: npt.NDArray[np.float32]
-
-    @property
-    def obs_status(self) -> List[str]:
-        return ["ACTIVE" if v else "DEACTIVATED" for v in self.obs_mask]
+    name: str
+    value: float
+    std: float
```

Comment:
Why rename from `obs_std` to `std`?

Reply:
I removed the prefix as I figured it was more centered around observations. Could rename from `UpdateSnapshot` to `ObservationSnapshot` instead perhaps?

Reply:
It's really `ObservationAndResponseSnapshot` 😬